src/jumio-kyc/kyc.controller.ts

```diff
@@ -151,14 +151,29 @@ export class KycController {
     const jumioTransaction =
       await this.jumioTransactionService.findLatestOrThrow(user);
 
+    // If we have a latest attempt, override with that reasons failure
+    if (jumioTransaction.last_workflow_fetch && !eligibility.reason) {
+      const { failureUrl, failureMessage } =
+        await this.redemptionService.calculateStatus(
+          jumioTransaction.last_workflow_fetch as unknown as JumioTransactionRetrieveResponse,
+        );
+
```

Comment:
This should work ok, but it could get hairy if this conditional is tweaked int he future. Currently it does truthy checks on a `string | null` type that is at times `''`, maybe you can just change the `HELP_URLS` to be `= null` instead of `= ''`